### PR TITLE
Pol 258 aws untagged resources

### PR DESCRIPTION
**Pull request by KVivek03**
### Description

AWS Untagged Resources Policy

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: compliance/aws/untagged_resources/aws_untagged_resources.pt [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+###############################################################################
+
+policy "policy_untagged_resources" do
+  validate $ds_filter_resources do
+    summary_template "AWS untagged resources list"
+    escalate $esc_email
+    escalate $esc_add_tags
+    check eq(size(data),0)
+    export do
+      resource_level true
```
Comment (cdwilhelm) [inline compliance/aws/untagged_resources/aws_untagged_resources.pt:206]: Is it possible to include the resource type? ie. instance, volume etc?

File: compliance/aws/untagged_resources/aws_untagged_resources.pt [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+      field "id" do
+        label "Resource ID"
+      end
+      field "region" do
+        label "Region"
+      end
```
[the remark this answers is not in the document]
Comment (KVivek03) [reply compliance/aws/untagged_resources/aws_untagged_resources.pt]: made the change

File: compliance/aws/untagged_resources/aws_untagged_resources.pt (hunk 1 above)
Comment (KVivek03) [reply compliance/aws/untagged_resources/aws_untagged_resources.pt:210]: added the resource type column

File: compliance/aws/untagged_resources/aws_untagged_resources.pt [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+      field "id" do
+        label "Resource ID"
+      end
+      field "resource_type" do
+        label "Resource Type"
+      end
+      field "available_tags" do
+        label "Available Tags"
```
Comment (cdwilhelm) [inline compliance/aws/untagged_resources/aws_untagged_resources.pt:221]: Available Tags seems like the wrong phrase.  Consider Existing Tags

Comment (KVivek03) [reply compliance/aws/untagged_resources/aws_untagged_resources.pt]: made the change

File: compliance/aws/untagged_resources/aws_untagged_resources.pt [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+
+escalation "esc_email" do
+  automatic true
+  label "Sends Email"
+  description "Sends Incident Email"
+  email $param_email
+end
+
+escalation "esc_add_tags" do
+  automatic false
```
Comment (cdwilhelm) [inline compliance/aws/untagged_resources/aws_untagged_resources.pt:190]: Missing the label and description

Comment (KVivek03) [reply compliance/aws/untagged_resources/aws_untagged_resources.pt:190]: added the label and the description.

**Comment by cdwilhelm** [comment]
@KVivek03    this PR needs to be updated with the automatic actions parameter

**Comment by KVivek03** [comment]
Hi @cdwilhelm , since the escalation block requires user input at the time of applying action so i think we need to leave escalation block as false only. Please let me know your thoughts. Thanks

**Comment by cdwilhelm** [comment]
> Hi @cdwilhelm , since the escalation block requires user input at the time of applying action so i think we need to leave escalation block as false only. Please let me know your thoughts. Thanks

@KVivek03 yes, you are right.  We should leave this one as false for now.

File: compliance/aws/untagged_resources/aws_untagged_resources.pt (hunk 4 above)
Comment (cdwilhelm) [reply compliance/aws/untagged_resources/aws_untagged_resources.pt:190]: @KVivek03   Can you please check the label?  The escalation/action label is probably better stated as Add Tags.

Comment (KVivek03) [reply compliance/aws/untagged_resources/aws_untagged_resources.pt:190]: modified the policy

File: nppBackup/README.md.2020-06-25_120237.bak [hunk 5]
```diff
@@ -0,0 +1,285 @@
+# Flexera CMP Policy Templates
+
```
[the remark this answers is not in the document]
Comment (KVivek03) [reply nppBackup/README.md.2020-06-25_120237.bak]: made the change

**Pull request merged**